app/services/invoice_pdf_generator.rb

```diff
@@ -179,8 +182,15 @@ def margin_bottom(lines)
     [0, 0, lines * BASE_FONT_SIZE]
   end
 
-  def format_cents(cents)
-    "#{format('%.2f', cents.to_f / 100)}€"
+  def to_money(value)
+    return value if value.is_a?(Money)
+    return Money.new(value['cents'], value['currency_iso']) if value.is_a?(Hash)
+
+    value
+  end
+
+  def format_money(money)
+    money.format(format: '%n%u')
```

Comment:
Until we understand how to use Rails locales and we can just use `money.format`, maybe use `format: '%n %u'` with a space? 😅

Reply:
It seems that money v7 (released the day you opened the PR 😅) and available through money-rails v2 does work correctly with I18n locale backend, so we could use a simple `.format` and let our locale rules do their magic.